src/components/SecondaryPanes/BreakpointItem.js

```diff
@@ -30,6 +34,41 @@ function getBreakpointLocation(source, line, column) {
 }
 
 class BreakpointItem extends Component<Props> {
+  editor: SourceEditor;
+
+  componentDidMount() {
+    this.setupEditor();
+  }
+
+  componentWillUnmount() {
+    this.editor.destroy();
+  }
+
+  shouldComponentUpdate(nextProps: Props) {
+    // This is needed, IMO, so we don't need to create
+    // loads of CM instances for no reason
+    return this.props.breakpoint != nextProps.breakpoint;
```

Comment:
CodeMirror seems like it could be a taxing tool, and since the other BreakpointItem props shouldn't change, I thought this could be useful.